.gitignore

```diff
@@ -1,5 +1,18 @@
-/target/
-/.settings/
-/.classpath
-/.project
-/bot.properties
+# Maven related
+target/
+
+# Javadoc dir
+javadocs/
+
+# IntelliJ Idea related
+.idea/
+*.iml
+
+# Eclipse related
+.settings/
+.classpath
+.project
+
+# Miscellenous
+*.properties
```

Comment:
I think the individual .properties files should be specified here instead of all .properties files.